### PR TITLE
Toast: add Vue and React code snippets

### PR DESCRIPTION
**Pull request by ksercs**

### CODE REVIEW

File: concepts/05 Widgets/Toast/00 Overview.md [hunk 1]
```diff
@@ -28,12 +28,58 @@ The **Toast** widget does not need to be created on the page before it can be sh
...
+##### React
+
+    import React from 'react';
+    import 'devextreme/dist/css/dx.common.css';
+    import 'devextreme/dist/css/dx.light.css';
+
+    import notify from "devextreme/ui/notify";
+
+    class App extends React.Component {
+        constructor(props) {
```
Comment (RomanTsukanov) [inline concepts/05 Widgets/Toast/00 Overview.md]: Don't declare a constructor if nothing happens in it.

File: concepts/05 Widgets/Toast/05 Show and Hide the Toast/01 API.md [hunk 2]
```diff
@@ -18,10 +18,10 @@ To show or hide the **Toast** programmatically, call the [show()](/api-reference
         });
     });
 
-With Angular, AngularJS, or Knockout, use a different technique. Bind the [visible](/api-reference/10%20UI%20Widgets/dxOverlay/1%20Configuration/visible.md '/Documentation/ApiReference/UI_Widgets/dxToast/Configuration/#visible') property of the **Toast** widget to a component property (in Angular), a scope property (in AngularJS), or an observable variable (in Knockout). After that, change this property or variable, and the **Toast** will appear or disappear.
+With Angular, Vue, or React, use a different technique. Bind the [visible](/api-reference/10%20UI%20Widgets/dxOverlay/1%20Configuration/visible.md '/Documentation/ApiReference/UI_Widgets/dxToast/Configuration/#visible') property of the **Toast** widget to a component property. After that, change this property or variable, and the **Toast** will appear or disappear.
```
Comment (RomanTsukanov) [inline concepts/05 Widgets/Toast/05 Show and Hide the Toast/01 API.md]:
```suggestion
With Angular, Vue, or React, use a different technique. Bind the [visible](/api-reference/10%20UI%20Widgets/dxOverlay/1%20Configuration/visible.md '/Documentation/ApiReference/UI_Widgets/dxToast/Configuration/#visible') property of the **Toast** widget to a component property. After that, change this property, and the **Toast** will appear or disappear.
```

File: concepts/05 Widgets/Toast/05 Show and Hide the Toast/10 Events.md [hunk 3]
```diff
@@ -1,10 +1,10 @@
 To execute certain commands before or after the **Toast** was shown/hidden, handle the [showing](/api-reference/10%20UI%20Widgets/dxOverlay/4%20Events/showing.md '/Documentation/ApiReference/UI_Widgets/dxToast/Events/#showing'), [shown](/api-reference/10%20UI%20Widgets/dxOverlay/4%20Events/shown.md '/Documentation/ApiReference/UI_Widgets/dxToast/Events/#shown'), [hiding](/api-reference/10%20UI%20Widgets/dxOverlay/4%20Events/hiding.md '/Documentation/ApiReference/UI_Widgets/dxToast/Events/#hiding') or [hidden](/api-reference/10%20UI%20Widgets/dxOverlay/4%20Events/hidden.md '/Documentation/ApiReference/UI_Widgets/dxToast/Events/#hidden') event. If the event handling function is not going to be changed during the lifetime of the widget, assign it to the corresponding **on*EventName*** option. For example, in the following code, a handler of the **hidden** event is assigned to the [onHidden](/api-reference/10%20UI%20Widgets/dxOverlay/1%20Configuration/onHidden.md '/Documentation/ApiReference/UI_Widgets/dxToast/Configuration/#onHidden') option. This handler counts down from three replacing the message in the **Toast** at the same time.
```
Comment (RomanTsukanov) [inline concepts/05 Widgets/Toast/05 Show and Hide the Toast/10 Events.md]:
```suggestion
To execute certain commands before or after the **Toast** was shown/hidden, handle the [showing](/api-reference/10%20UI%20Widgets/dxOverlay/4%20Events/showing.md '/Documentation/ApiReference/UI_Widgets/dxToast/Events/#showing'), [shown](/api-reference/10%20UI%20Widgets/dxOverlay/4%20Events/shown.md '/Documentation/ApiReference/UI_Widgets/dxToast/Events/#shown'), [hiding](/api-reference/10%20UI%20Widgets/dxOverlay/4%20Events/hiding.md '/Documentation/ApiReference/UI_Widgets/dxToast/Events/#hiding') or [hidden](/api-reference/10%20UI%20Widgets/dxOverlay/4%20Events/hidden.md '/Documentation/ApiReference/UI_Widgets/dxToast/Events/#hidden') event. If the event handling function is not going to be changed during the lifetime of the widget, assign it to the corresponding **on*EventName*** option:
```

File: concepts/05 Widgets/Toast/05 Show and Hide the Toast/10 Events.md [hunk 4]
```diff
@@ -1,10 +1,10 @@
...
 
 ---
-#####jQuery
+##### jQuery
```
Comment (RomanTsukanov) [inline concepts/05 Widgets/Toast/05 Show and Hide the Toast/10 Events.md:4]: This code sample is really strange. Let's use a generic example instead. Something like:

```
onHiding: () => {
    // Handler of the "hiding" event
},
onHidden: () => {
    // Handler of the "hidden" event
},
```

and so on.

File: concepts/05 Widgets/Toast/10 Customize the Content.md [hunk 5]
```diff
@@ -155,6 +253,122 @@ If you need to render different templates depending on a specific condition, def
...
+            };
+
+            this.onClick = this.onClick.bind(this);
+            this.onHiding = this.onHiding.bind(this);
+        }
+
+        onClick() {
+            this.setState({
+                isVisible: true,
+                renderContent: this.state.renderContent === greenTemplate ? blueTemplate : greenTemplate
```
Comment (RomanTsukanov) [inline concepts/05 Widgets/Toast/10 Customize the Content.md:334]: Looks interesting :) Does it actually work?

**Review by RomanTsukanov — CHANGES_REQUESTED** [review]

File: concepts/05 Widgets/Toast/00 Overview.md (hunk 1 above)
Comment (ksercs) [reply concepts/05 Widgets/Toast/00 Overview.md]: fixed

File: concepts/05 Widgets/Toast/05 Show and Hide the Toast/01 API.md (hunk 2 above)
Comment (ksercs) [reply concepts/05 Widgets/Toast/05 Show and Hide the Toast/01 API.md]: fixed

File: concepts/05 Widgets/Toast/05 Show and Hide the Toast/10 Events.md (hunk 3 above)
Comment (ksercs) [reply concepts/05 Widgets/Toast/05 Show and Hide the Toast/10 Events.md]: fixed

File: concepts/05 Widgets/Toast/10 Customize the Content.md (hunk 5 above)
Comment (ksercs) [reply concepts/05 Widgets/Toast/10 Customize the Content.md:334]: yeap, I test all non-trivial snippets

File: concepts/05 Widgets/Toast/05 Show and Hide the Toast/10 Events.md (hunk 4 above)
Comment (ksercs) [reply concepts/05 Widgets/Toast/05 Show and Hide the Toast/10 Events.md:4]: fixed

**Review by RomanTsukanov — APPROVED** [review]

**Review by AlexJameson — APPROVED** [review]

**Pull request merged**